src/zenml/models/v2/core/device.py

```diff
@@ -48,6 +174,11 @@ class OAuthDeviceBaseModel(BaseModel):
     status: OAuthDeviceStatus = Field(
         description="The status of the OAuth2 device."
     )
+
+
+class OAuthDeviceResponseMetadata(UserScopedResponseMetadata):
```

Comment:
Could you please move `status`, `ip_address`, `hostname` and `os` to the body ? I think these will be used in the dashboard as well. They are already used in the CLI.

Reply:
The dashboard will usually fetch the hydrated version of a model in case it displays a detail screen of that model I assume?

I think we need some kind of common understanding to decide what usually goes into the body/metadata, as it's not entirely clear to me yet. If the information is useful when `listing` the entity, I assume it makes sense to include it in the body to make it easy for the dashboard?

Reply:
That's what I meant: used in the dashboard for listing. The same is true for the CLI.

The hidden danger here is the CLI might fetch some attribute to display a list of X and accidentally calling the API 20 more times to display 20 items.

Reply:
`status` was already in the body. `ip_adress`, `hostname`, and `os` have been moved to the body.